src/main/java/duke/command/ExitCommand.java

```diff
@@ -0,0 +1,30 @@
+package duke.command;
+
+import duke.storage.Storage;
+import duke.task.TaskList;
+import duke.ui.Ui;
+
+/**
+ * A class represents ExitCommand
+ */
+public class ExitCommand extends Command {
+    /**
+     * Calls ui to print a farewell message.
+     * @param tasks Task list given.
+     * @param ui User interface class object.
+     * @param storage Storage path that is going to be updated.
+     */
+    @Override
+    public void execute(TaskList tasks, Ui ui, Storage storage) {
+        ui.sayGoodBye();
+    }
+
+    /**
+     * Returns true.
```

Comment:
Should describe what the function is intended to achieve, instead of stating how.
Perhaps, "Passes instruction to main routine to exit the program"

Reply:
Thanks!